core/camel-endpointdsl/src/main/java/org/apache/camel/builder/endpoint/dsl/BrowseEndpointBuilderFactory.java

```diff
@@ -209,46 +209,6 @@ default AdvancedBrowseEndpointConsumerBuilder synchronous(
         default AdvancedBrowseEndpointProducerBuilder advanced() {
             return (AdvancedBrowseEndpointProducerBuilder) this;
         }
-        /**
```

Comment:
You have to rebuild the project fully, that option should be there https://github.com/apache/camel/blob/master/core/camel-endpointdsl/src/main/java/org/apache/camel/builder/endpoint/dsl/BrowseEndpointBuilderFactory.java#L213